### ISSUE TITLE
Option to order the groups and properties by name

### ISSUE THREAD

**Issue by nandorholozsnyak** [rodnansol/spring-configuration-property-documenter#88]
**Is your feature request related to a problem? Please describe.**
The groups and properties are having a fixed order that the developers are not able to customize, it would be great if there would be two different options:
- orderGroup
- orderProperties

**Comment by nandorholozsnyak** [comment]
Hello @roded 

Can you show me some examples and expected outcome? Thank you.

**Comment by roded** [comment]
Hi @nandorholozsnyak, sorry I don't have examples to share, but I can try to explain the issue.
I have 2 Spring Boot starters which share a configuration namespace, e.g.: `my.component.config`.
When creating the documentation for a project which depends on these two starters, the generated configuration document does not sort the configuration properties by name, so `my.component.config.b` does not necessarily follow `my.component.config.a`. Hope that makes sense. 

Long story short, I think that spring-configuration-property-documenter should sort the properties lexicographically.
